samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -102,4 +109,34 @@ public SemanticKernelController(IServiceProvider serviceProvider, IConfiguration
 
         return this.Ok(new AskResult { Value = result.Result, Variables = result.Variables.Select(v => new KeyValuePair<string, string>(v.Key, v.Value)) });
     }
+
+    private async Task RegisterOpenApiSkills(PluginAuthHeaders pluginAuthHeaders, Kernel kernel)
+    {
+        if (pluginAuthHeaders.GithubAuthentication != null)
+        {
+            var authenticationProvider = new BearerAuthenticationProvider(() => { return Task.FromResult(pluginAuthHeaders.GithubAuthentication); });
+            this._logger.LogInformation("Registering GitHub Skill, token {0}", pluginAuthHeaders.GithubAuthentication);
+
+            var skill = await kernel.ImportOpenApiSkillFromResourceAsync(SkillResourceNames.GitHub,
```

Comment:
Can we switch to using either 
 kernel.ImportOpenApiSkillFromFile
or kernel.ImportOpenApiSkillFromUrlAsync instead?
Importing from a file or URL is more straight forward and requires fewer modifications to the code files. I also envision developers just being able to point to a URL where there schema exists and taking it from there especially if they are working with 3rd party schemas.
Using ImportOpenApiSkillFromFile is more appropriate here than the URL just because we want to showcase a limited set of operations that we have verified work.
This should also make it so that you dont have to touch files like SkillResourceName.cs

Reply:
Done